### PR TITLE
WD-11589 - refactor: remove nested components in StatusGroup

### PR DESCRIPTION
**Pull request by vladimir-cucu**
## Done

- Remove nested components in `StatusGroup`.

## Details

- https://warthogs.atlassian.net/browse/WD-11589

### CODE REVIEW

File: src/components/ModelTableList/StatusGroup/StatusGroup.test.tsx [hunk 1]
```diff
@@ -177,50 +164,19 @@ describe("StatusGroup", () => {
     );
   });
 
-  it("displays links to blocked apps and units", async () => {
-    vi.useFakeTimers();
+  it("displays links to blocked apps", async () => {
```
Comment (vladimir-cucu) [inline src/components/ModelTableList/StatusGroup/StatusGroup.test.tsx:167]: I think we still need to test that these links get rendered, however, I've removed the bit regarding the Tooltip testing, as the tests for it are present in `WarningMessage.test.tsx`.

File: src/components/ModelTableList/StatusGroup/WarningMessage/WarningMessage.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+import urls from "urls";
+import { getUserName } from "utils";
+
+type Props = {
+  model: ModelData;
+};
+
+/**
+  Warning message for the model name cell.
+  @param model The full model data.
```
Comment (huwshimi) [inline src/components/ModelTableList/StatusGroup/WarningMessage/WarningMessage.tsx:17]: This component doesn't have a `model` param anymore (the `model` arg is part of the props object). If you would like this comment to appear as a hint when using the component you could move this comment to the props type.

```typescript
type Props = {
  /**
   * The full model data.
   */
  model: ModelData;
};
```

**Review by huwshimi — APPROVED** [review]
Just one tiny suggestion about the doc string.

File: src/components/ModelTableList/StatusGroup/WarningMessage/WarningMessage.tsx (hunk 2 above)
Comment (vladimir-cucu) [reply src/components/ModelTableList/StatusGroup/WarningMessage/WarningMessage.tsx]: Fixed! Thanks for the review.

**Pull request merged**